File: src/main/java/net/arsentic/library/PluginUtils.java
package net.arsentic.library;

import org.apache.commons.lang.StringUtils;
import org.bukkit.inventory.ItemStack;

import java.text.SimpleDateFormat;
import java.util.Date;

// Damn i hate static but like, gotta do it
public final class PluginUtils {

    public static String formatTime(Long time) {
        return new SimpleDateFormat("HH:mm dd/m/yyy").format(new Date(time));
    }

    public static String transformName(ItemStack itemStack) {
        return StringUtils.capitalize(itemStack.getType().name().replace("_", " ").toLowerCase());
    }

    public static boolean isNumber(String string) {
        try {
            Integer.parseInt(string);
        } catch (NumberFormatException ex) {
            return false;
        }

        return true;
    }
}

File: src/main/java/net/arsentic/library/PluginUtils.kt
package net.arsentic.library

import org.bukkit.inventory.ItemStack
import java.text.SimpleDateFormat
import java.util.*

object PluginUtils {

    @JvmStatic
    fun formatTime(long: Long): String {
        return SimpleDateFormat("HH:mm dd/m/yyyy").format(Date(long))
    }

    @JvmStatic
    fun transformName(itemStack: ItemStack): String {
        return itemStack.type.name.replace("_", " ").toLowerCase().capitalize()
    }
}
